vng_api_common/inspectors/fields.py

```diff
@@ -124,3 +125,19 @@ def process_result(self, result, method_name, obj, **kwargs):
         schema.x_nullable = True
 
         return result
+
+
+class HistoryURLFieldExtension(OpenApiSerializerFieldExtension):
+    target_class = "rest_framework.fields.SerializerMethodField"
+    match_subclasses = True
+    method_name = "create_custom_urls"
+
+    def map_serializer_field(self, auto_schema, direction):
+        default_schema = auto_schema._map_serializer_field(
+            self.target, direction, bypass_extensions=True
+        )
+        if self.target._kwargs.get("method_name", "") == self.method_name:
```

Comment:
```suggestion
        if self.target.__class__.__name__ == "HistoryURLField":
```
Vooral om de situatie te voorkomen dat een ander veld ook dezelfde keyword argument gebruikt (en onterecht de description krijgt)